src/components/TextArea/TextArea.tsx

```diff
@@ -1,21 +1,27 @@
 import React, { useState } from 'react';
 
-import scss from './TextArea.module.scss';
+import scss from '~components/TextArea/TextArea.module.scss';
 
-function CustomTextArea() {
+import { ICustomTextAreaProps } from './TextArea.types';
+
+function CustomTextArea({ onTextChange }: ICustomTextAreaProps) {
   const [inputValue, setInputValue] = useState('');
 
   const handleInputChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
-    setInputValue(event.target.value);
+    const newValue = event.target.value;
+    setInputValue(newValue);
+    onTextChange && onTextChange(newValue);
```

Comment:
Aqui sem a desestruturação, da pra fazer isso:

```suggestion
    props?.onTextChange(newValue);
```

Reply:
Pronto.